endpoints/data_objects/src/main.cpp

```diff
@@ -826,57 +944,59 @@ namespace
 					return _sess_ptr->send(irods::http::fail(res, http::status::bad_request));
 				}
 
-				static const auto buffer_size =
+				static const auto max_number_of_bytes_per_write =
 					irods::http::globals::configuration()
 						.at(json::json_pointer{"/irods_client/buffer_size_in_bytes_for_write_operations"})
```

Comment:
Consider renaming `/irods_client/buffer_size_in_bytes_for_write_operations` config option.

It is not a buffer size. It is the max number of bytes to write to iRODS in a single call.

Reply:
Renamed to `max_number_of_bytes_per_write_operation`.

Uses the same naming scheme as the option for read operations.